lib/cb/responses/job/search_v3.rb

```diff
@@ -0,0 +1,28 @@
+module Cb
+  module Responses
+    module Job
+      class SearchV3 < ApiResponse
+
+        protected
+
+        def validate_api_hash
+          required_response_field(root_node, response)
+        end
+
+        def hash_containing_metadata
```

Comment:
My mistake.  We do have a search metadata node.  Provides fields like search_locations, results_are_grouped, and job search_evidence_id.